app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java

```diff
@@ -75,7 +75,7 @@ public enum SettingsEnum {
     SWIPE_OVERLAY_TIMEOUT("revanced_swipe_overlay_timeout", 500L, ReturnType.LONG),
     SWIPE_OVERLAY_TEXT_SIZE("revanced_swipe_overlay_text_size", 22f, ReturnType.FLOAT),
     SWIPE_OVERLAY_BACKGROUND_ALPHA("revanced_swipe_overlay_background_alpha", 127, ReturnType.INTEGER),
-    SWIPE_MAGNITUDE_THRESHOLD("revanced_swipe_magnitude_threshold", 30f, ReturnType.FLOAT),
+    SWIPE_MAGNITUDE_THRESHOLD("revanced_swipe_magnitude_threshold", 15f, ReturnType.FLOAT),
```

Comment:
This should not be changed by this PR.

Reply:
my bad...there u go https://github.com/revanced/revanced-integrations/pull/128